tasks.c

```diff
@@ -7274,7 +7274,8 @@ static void prvResetNextTaskUnblockTime( void )
         int iSnprintfReturnValue;
         BaseType_t xOutputBufferFull = pdFALSE;
         UBaseType_t uxArraySize, x;
-        configRUN_TIME_COUNTER_TYPE ulTotalTime, ulStatsAsPercentage;
+        configRUN_TIME_COUNTER_TYPE ulTotalTime = 0;
+        configRUN_TIME_COUNTER_TYPE ulStatsAsPercentage;
```

Comment:
Should this also receive an initial value?

Reply:
ulStatsAsPercentage is assigned a value [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/tasks.c#L7335), before it is read, so it does not cause this MISRA violation even if it is not given an initial value.